Formula/influxdb@1.rb

```diff
@@ -0,0 +1,96 @@
+class InfluxdbAT1 < Formula
+  desc "Time series, events, and metrics database"
+  homepage "https://influxdata.com/time-series-platform/influxdb/"
+  url "https://github.com/influxdata/influxdb/archive/v1.8.4.tar.gz"
+  sha256 "9f2c135c8f9f50ca469196e6b4e575e26f1a338538788e71b664212e03b4df7b"
+  license "MIT"
+
+  # The regex below omits a rogue `v9.9.9` tag that breaks version comparison.
+  livecheck do
+    url :stable
+    regex(/^v?((?!9\.9\.9)1.+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^v?(1(?:\.\d+)+)$/i)
```
Sorry, messed up that regex. This one should work. Squash your commits together when you make this change, please. Thanks!

Reply:
Thank you very much for help with this MR!